test/e2etest/kubetest/setup/setup.go

```diff
@@ -0,0 +1,123 @@
+package setup
+
+import (
+	"fmt"
+	"github.com/gardener/test-infra/test/e2etest/config"
+	"github.com/gardener/test-infra/test/e2etest/kubetest"
+	"github.com/gardener/test-infra/test/e2etest/util"
+	"github.com/mholt/archiver"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+)
+
+var k8sOutputBinDir string = filepath.Join(config.KubernetesPath, "_output/bin")
+
+func Setup() error {
```

Comment:
maybe we should also check here if kubectl etc is ready and installed.
Then we would have all prerequisites in one place